source/MonoGame.Extended.Gui/FNA.Extended.Gui.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>A GUI system for FNA written from the ground up to makes FNA more awesome.</Description>
+    <PackageTags>fna gui</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Remove="MonoGame.Framework.DesktopGL" />
+
+    <ProjectReference Include="..\..\..\FNA\FNA.NetStandard.csproj">
```

Comment:
See note from Kni.Extended PR here https://github.com/craftworkgames/MonoGame.Extended/pull/892#discussion_r1642013504

If we can do something similar with FNA